app/models/node.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: nodes
+#
+#  id             :bigint           not null, primary key
+#  description    :text
+#  name           :string           not null
+#  number         :integer          not null
+#  plate          :string           not null
+#  reference_code :uuid             not null
+#  relative_age   :integer          not null
+#  seal           :string(3)        not null
+#  serie          :string(3)        not null
+#  size           :float            not null
+#  status         :string           not null
+#  time_slot      :string           not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  category_id    :bigint           not null
+#
+# Indexes
+#
+#  index_nodes_on_category_id     (category_id)
+#  index_nodes_on_plate           (plate) UNIQUE
+#  index_nodes_on_reference_code  (reference_code) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (category_id => categories.id)
+#
+require "uuidtools"
+
+class Node < ApplicationRecord
+  belongs_to :category
+  has_and_belongs_to_many :tags
+
+  has_enumeration_for :status, with: NodeStatusEnum, create_helpers: true
+  has_enumeration_for :time_slot, with: TimeSlotEnum, create_helpers: true
+
+  validates :name, :seal, :serie, :plate, :status, :number, :size, :reference_code, :time_slot, :relative_age, presence: true
+  validates :plate, uniqueness: true
```

Comment:
```suggestion
  validates :plate, :reference_code, uniqueness: true
```
Why not same line for different fields with same? like all the document